frontend/apps/reader/modules/readermenu.lua

```diff
@@ -145,6 +145,26 @@ function ReaderMenu:setUpdateItemTable()
                 }
             }
         }
+        elseif Device:isKindle() and Device:supportsScreensaver() then
+        self.menu_items.screensaver = {
+            text = _("Screensaver"),
+            sub_item_table = {
+                {
+                    text = _("Use book's cover as screensaver"),
```

Comment:
Can we keep this text consistent with what's in filemanager menu? It should be `_("Use last book's cover as screensaver")`.